include/QtNodes/internal/GraphicsView.hpp

```diff
@@ -77,5 +84,6 @@ public Q_SLOTS:
   QAction* _deleteSelectionAction;
 
   QPointF _clickPos;
+  QPointF _scaleRange;
```

Comment:
`QPointF` usually stands for some position. I would stick to `std::pair` 
Or maybe even `struct ScaleRange { double minimum, double maximum }`